### PR TITLE
gui-wm/hyprland: add 0.35.0-r1

### PR DESCRIPTION
**Pull request by MrRoy**
sys-auth/polkit is no longer necessary for hyprpm in hyprland 0.35.0

Additionally, whilst I was syncing the live ebuild, I noticed that now hyprlan**d** master depends on hyprlan**g**, a library used to parse the hypr config language. That package is currently in guru since it's already used by other hypr-related packages (for instance, hyprpaper), but now it is required by hyprland-9999 and will most likely be required for the next hyprland version, so this PR adds that ahead of the next release.

### CODE REVIEW

**Comment by MrRoy** [comment]
CC: @dlan17 @parona-source

**Comment by parona-source** [comment]
https://github.com/hyprwm/Hyprland/commit/3656045ad88717fbc079c1ebb44c6f7d2ed85cf2 include this in commit message

File: dev-libs/hyprlang/hyprlang-0.4.0.ebuild [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+HOMEPAGE="https://github.com/hyprwm/hyprlang"
+SRC_URI="https://github.com/hyprwm/${PN}/archive/v${PV}.tar.gz -> ${P}.gh.tar.gz"
+
+LICENSE="GPL-3"
+SLOT="0"
+KEYWORDS="~amd64"
+IUSE="test"
+RESTRICT="!test? ( test )"
+
+BUILD_DIR="${S}/build"
```
Comment (parona-source) [inline dev-libs/hyprlang/hyprlang-0.4.0.ebuild:18]: what is this?

File: dev-libs/hyprlang/hyprlang-0.4.0.ebuild [hunk 2]
```diff
@@ -0,0 +1,25 @@
...
+inherit cmake
+
+DESCRIPTION="Official implementation library for the hypr config language"
+HOMEPAGE="https://github.com/hyprwm/hyprlang"
+SRC_URI="https://github.com/hyprwm/${PN}/archive/v${PV}.tar.gz -> ${P}.gh.tar.gz"
+
+LICENSE="GPL-3"
+SLOT="0"
+KEYWORDS="~amd64"
+IUSE="test"
```
Comment (parona-source) [inline dev-libs/hyprlang/hyprlang-0.4.0.ebuild:15]: What is test use flag doing here?

File: dev-libs/hyprlang/hyprlang-0.4.0.ebuild [hunk 3]
```diff
@@ -0,0 +1,25 @@
...
+RESTRICT="!test? ( test )"
+
+BUILD_DIR="${S}/build"
+
+src_prepare() {
+	default
+
+	sed "s/\<lib\>/$(get_libdir)/" -i CMakeLists.txt hyprlang.pc.in
```
Comment (parona-source) [inline dev-libs/hyprlang/hyprlang-0.4.0.ebuild:23]: Doesn't appear to be necessary https://github.com/hyprwm/hyprlang/commit/25da0804b00fffeee17463afd146711b4a05e77b

**Comment by parona-source** [comment]
After this gets merged remember to request riscv keywords for hyprland, no dropped keywords ideally for hyprland when the bump comes.

**Comment by MrRoy** [comment]
@parona-source all fixed

**Pull request closed without merging**